index.js

```diff
@@ -35,6 +35,7 @@ const slugify = (string, options) => {
 
 	options = {
 		separator: '-',
+		leadingUnderscore: false,
```

Comment:
Super nitpick, but it should be in the same order here as in the readme.

Reply:
Just commited all the changes!